### PR TITLE
 TFW-3974 - Navigate on bookmark selection

### PR DESCRIPTION
**Pull request by jasonhale**

### CODE REVIEW

File: assets/js/demo/components/universalSearch.js [hunk 1]
```diff
@@ -57,11 +85,18 @@ export default class UniversalSearchDemo extends Component {
...
 
-        if (location.search) {
+        if (location.search.length > 0) {
             const { bookmarks } = this.state;
+            const id = location.search.split('id=')[1];
+
+            const optionedbookmark = this.bookmarkOptions.find(b => b.value === id);
 
-            if (!bookmarks.activeBookmark) {
+            console.log(optionedbookmark);
```
Comment (troyblank) [inline assets/js/demo/components/universalSearch.js]: forgot to remove this :)

File: components/universalSearch/bookmarking/header.js [hunk 2]
```diff
@@ -10,17 +10,13 @@ export default class BookMarkHeader extends PureComponent {
...
         return (
           <div className={'bookmark-header'}>
             <IconBookmark size={'medium'} />
             <HamburgerNavigation label={'Bookmarks'} onOpen={this.onSelectorOpenHandler}>
               <BookmarkSelector
-                options={options}
                 ariaID={'Bookmarks'}
                 ref={selector => (this.selector = selector)}
-                manageBookmarkURL={manageBookmarkURL}
               />
```
Comment (troyblank) [inline components/universalSearch/bookmarking/header.js]: This has caused a weird coverage issue with ref 🤷‍♂

File: assets/js/demo/components/universalSearch.js (hunk 1 above)
Comment (jasonhale) [reply assets/js/demo/components/universalSearch.js]: sunnuvab...

File: components/universalSearch/bookmarking/header.js (hunk 2 above)
Comment (jasonhale) [reply components/universalSearch/bookmarking/header.js]: Coverage has been all over the place.  Will check this out.

Comment (troyblank) [reply components/universalSearch/bookmarking/header.js]: Yea using enzyme has not been great as of late, maybe when we get less busy we can talk about a transition to something like jest or something :/

Comment (jasonhale) [reply components/universalSearch/bookmarking/header.js]: SO... modified the component so coverage would be _correct_.

**Comment by jasonhale** [comment]
Updated this PR to allow for actual coverage of the problematic function in bookmark selector component.

Also, options for the bookmarks selector are being build component-side, not in RCI and passed.

**Pull request merged**